src/dc_imex.rs

```diff
@@ -962,18 +883,13 @@ unsafe fn export_self_keys(context: &Context, dir: *const libc::c_char) -> bool
 /*******************************************************************************
  * Classic key export
  ******************************************************************************/
-unsafe fn export_key_to_asc_file(
-    context: &Context,
-    dir: *const libc::c_char,
-    id: Option<i64>,
-    key: &Key,
-) -> bool {
+fn export_key_to_asc_file(context: &Context, dir: &str, id: Option<i64>, key: &Key) -> bool {
```

Comment:
I would change `dir` to `impl AsRef<Path>` if you are already changing it.

Reply:
done